src/fsharp/FSharp.Core/prim-types.fsi

```diff
@@ -1097,6 +1097,14 @@ namespace Microsoft.FSharp.Core
             //[<CompilerMessage("This function is for use by compiled F# code and should not be used directly", 1204, IsHidden=true)>]
             val inline GetString : source:string -> index:int -> char
 
+            /// <summary>A compiler intrinsic that gets character from a string</summary>
+            [<CompilerMessage("This function is for use by compiled F# code and should not be used directly", 1204, IsHidden=true)>]
+            val inline GetStringChar : source:string -> index:int -> char
```

Comment:
I am looking for alternatives to add these methods to prim-types.fsi. The issue is that in the optimizer there is not so much support for find the correct overloads. In the type-checker it was possible to find the proper overload but migrating that code to the optimizer proved difficult.